homu/main.py

```diff
@@ -216,7 +216,12 @@ def refresh(self):
         self.body = issue.body
 
     def fake_merge(self, repo_cfg):
```

Comment:
Oh sorry, `git grep exempt` showed me.  The answer is this is also called by `report_build_res()`.